.github/workflows/release.yml

```
@@ -48,28 +52,20 @@ jobs:
           password: ${{ secrets.GITHUB_TOKEN }}
       - name: ${{ env.POETRY_VERSION }}-python-${{ matrix.python }}
         uses: docker/build-push-action@v2
+        if: ${{ matrix.variant == "buster" }}
```

Instead of doing this, I suggest:

1. Including extra values indicating the default Python version and variant[^1]
2. Building:
  a. First, the full tag: `${{ env.POETRY_VERSION }}-python-${{ matrix.python }}-${{ matrix.variant }}`
  b. Then, the shortcut: `${{ env.POETRY_VERSION }}-python-${{ matrix.python }}`

```suggestion
        if: ${{ matrix.is_default_variant }}
```

[^1]: https://github.com/withlogicco/docker-poetry/pull/3/files#diff-87db21a973eed4fef5f32b267aa60fcee5cbdf03c67fafdc2a9b553bb0b15f34R44

---

Nice idea 🎉